README.md

```diff
@@ -1,2 +1,173 @@
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello-World<a name="Creating Microverse Hello-worl project "></a>
+
+
+**Hello World** is a project build with the knowledge of Html and CSS that has a header.
+
+## 🛠 Built With <a name="HTML&CSS"></a>
+
+### Tech Stack <a name="HTML,CSS and LINTERS"></a>
+
+<details>
+### Key Features <a name="key-features"></a>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https:w3schools.com">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https:w3schools.com">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>LINTERS</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [NOW AVAILABLE AT THE MOMENT]
+
+## 💻 Getting Started <a name="https://github.com/shurichmond/Hello-World"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1** Richmond Shu Nfortayong
+
+- GitHub: [@githubhandle](https://github.com/shurichmond)
+- Twitter: [@twitterhandle](https://twitter.com/shurichmond85)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shu-nfortayong-richmond-10a28620a/)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Contact form and design improvements]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/shurichmond/Hello-World).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
```

Comment:
- [ ] Consider removing the placeholder text and customizing this section as the previous reviewer suggested. 💯 💯 Here is a sample text you can add here:
- I would like to thank Microverse for providing the guideline for this project, Also also the resources to  make it a success